### PR TITLE
Additions, Fixes & Updations to make ocs-operator-ci

### PR DESCRIPTION
**Pull request by malayparida2000**

### CODE REVIEW

**Review by iamniting — CHANGES_REQUESTED** [review]

File: controllers/storagecluster/external_resources_test.go [hunk 1]
```diff
@@ -267,7 +267,7 @@ func updateNamedResourceInArray(extArr []ExternalResource, extRsrc ExternalResou
 }
 
 func generateRandomPort(minPort, maxPort int) int {
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```
[the remark this answers is not in the document]
Comment (malayparida2000) [reply controllers/storagecluster/external_resources_test.go:270]: Updated

**Comment by iamniting** [comment]
@malayparida2000 Can you take a look at the failures?

**Review by iamniting — APPROVED** [review]

**Pull request merged**